operational/flexera/custom_branding/flexera_custom_branding.pt

```diff
@@ -0,0 +1,359 @@
+  default "No Change"
+end
+
+parameter "param_marketing_support_plan_hide" do
+  type "string"
+  category "All Customer Settings"
+  label "Marketing Support Plan Hide"
+  description "A boolean to determine show/hide of the Support Plan Information link on the landing page."
+  allowed_values "Yes", "No", "No Change"
+  default "No Change"
+end
+
+parameter "param_marketing_flexera_com_hide" do
+  type "string"
+  category "All Customer Settings"
+  label "Marketing Submit Ideas Hide"
+  description "A boolean to determine show/hide of the Flexera.com link on the landing page."
+  allowed_values "Yes", "No", "No Change"
+  default "No Change"
+end
+
+parameter "param_login_logo_image" do
+  type "string"
+  category "MSP Specific Settings"
+  label "Login Logo Image"
+  description "The web address of the image resource file location to use in place of the Flexera logo for unauthenticated pages like login, password reset, and so on."
+  default "No Change"
+end
+
+parameter "param_global_customer_support_link" do
+  type "string"
+  category "MSP Specific Settings"
+  label "Global Customer Support Link"
+  description "A URL override pointing to your own Customer Support link on the login page and elsewhere throughout the app."
+  default "No Change"
+end
+
+parameter "param_marketing_login_image" do
+  type "string"
+  category "MSP Specific Settings"
+  label "Marketing Login Image"
+  description "The web address of the image resource file location to use in place of the Flexera marketing section on the login page."
+  default "No Change"
+end
+
+parameter "param_marketing_login_cta_link" do
+  type "string"
+  category "MSP Specific Settings"
+  label "Marketing Login CTA Link"
+  description "A URL override pointing to your own Download Now call to action link on the login page."
+  default "No Change"
+end
+
+parameter "param_marketing_login_cta_text" do
+  type "string"
+  category "MSP Specific Settings"
+  label "Marketing Login CTA Text"
+  description "Plain text override of the Download Now call to action text displayed on the login page."
+  max_length 20
+  default "No Change"
+end
+
+parameter "param_marketing_login_eyebrow_text" do
+  type "string"
+  category "MSP Specific Settings"
+  label "Marketing Login Eyebrow Text"
+  description "Plain text to override of the eyebrow marketing text on the login page."
+  max_length 30
+  default "No Change"
+end
+
+parameter "param_marketing_login_blurb_text" do
+  type "string"
+  category "MSP Specific Settings"
+  label "Marketing Login Blurb Text"
+  description "Plain text to override of the marketing blurb on the login page."
+  max_length 150
+  default "No Change"
+end
+
+parameter "param_marketing_login_title_text" do
+  type "string"
+  category "MSP Specific Settings"
+  label "Marketing Login Title Text"
+  description "Plain text override of the marketing title text on the login page."
+  max_length 30
+  default "No Change"

```

Comment:
Typically, if the parameter is asking for optional information, we should set the default to an empty string and put something in the description like "Leave blank to not configure a Nav Logo Image."